packages/design-tokens/README.md

```diff
@@ -0,0 +1,38 @@
+# Kaizen Tokens
+Design tokens for all platforms.
+
+## About
+Design Tokens are the heart of every Design System. The tokens represented here are platform-agnostic (JSON), as this will help us contribute to & facilitate the maintenance of living style guides. This package defines all the option tokens in Kaizen.
+
+(An "option token" is a token like `$kz-color-wisteria-800: #35374A` that is one of the options in a palette and contains no information about _how_ it should be used, as opposed to a "decision token" like `$default-text-color: $kz-color-wisteria-800` which does contain information about how the token should be used.)
+
+In its current state this package supports Sass variables, generated from a JSON tokens file. In the near future we will be supporting more tokens. 
+
+**Please note** that the helpers in this package are specifically for accessing and using these design tokens. Component-specific helpers are best suited for kaizen-component-library. 
+
+## Installation
+```
+yarn add @cultureamp/kaizen-tokens
+```
+
+## Usage
+### Sass
+```
+## Note helper functions are provided
+@import "~@cultureamp/kaizen-design-tokens/lib/sass/[color/depth/layout/spacing/typography/helpers]";
+```
+
+### Javascript 
+```
+@import * as tokens from @cultureamp/kaizen-tokens/tokens/[color/depth/layout/spacing/typography]
```

Comment:
Should be `@cultureamp/kaizen-design-tokens/...`?

Reply:
Just had another idea on this – wonder if there's a way we could do what we did with the component library and use `index.js` barrel files to make an API that isn't coupled to a particular dir structure.

So people can just do
```scss
@import "~@cultureamp/kaizen-design-tokens/[color/depth/layout/spacing/typography/helpers].scss"
// no lib/sass/
```
```js
@import * as tokens from "@cultureamp/kaizen-design-tokens/[color/depth/layout/spacing/typography]"
// no tokens/
```

Reply:
Good catch @mbylstra, I'll update the docs. 

I wasn't sure what the future of tokens would look like/if we needed to support Less tokens in the future, so that's why I moved it to its own directory. I can flatten this to `"~@cultureamp/kaizen-design-tokens/sass/[...]`? Thoughts?

Reply:
Actually @didoesdigital had a comment here that I previously missed: 
https://github.com/cultureamp/kaizen-design-system/pull/93#discussion_r342850300

Thoughts on changing to `@cultureamp/design-tokens`, or even `@kaizen/design-tokens`?

Reply:
> I wasn't sure what the future of tokens would look like/if we needed to support Less tokens in the future, so that's why I moved it to its own directory. I can flatten this to "~@cultureamp/kaizen-design-tokens/sass/[...]? Thoughts?

I don't mean we would have to get rid of any of our internal dirs like `/lib/sass` or anything :)

I mean like what we've done with the component library, where we've got `index.ts` files that export everything and allow you to import directly from the top level dir without having to know anything about the structure beyond that:

![image](https://user-images.githubusercontent.com/6406263/69107168-a36e3d00-0ac4-11ea-9a11-2d38754664b5.png)

Reply:
Re: Thoughts on changing to `@cultureamp/design-tokens`, or even `@kaizen/design-tokens`?

I'd prefer `@kaizen/design-tokens` as long as time doesn't become an issue.

Reply:
> Because of that I think I prefer design-tokens without the kaizen- bit.
Agreed, I'll change this. 

Oh, I see what you mean. This works for javascript because you can import something specific (or everything) from a list of exports... but it doesn't work so well with Sass which makes everything global as soon as you import the file.

Reply:
Ah yeah, bummer. Maybe just having `/sass` is a good compromise then? Just to make it easy to remember if writing the import manually.

Reply:
If we leave `@cultureamp/kaizen-component-library`  the same, then removing `kaizen` from `@cultureamp/kaizen-design-tokens` seems inconsistent. Perhaps we should wait until if we decide to use `@kaizen` and then do a big refactor for both: eg (`@kaizen/component-library` and `@kaizen/design-tokens`)

Reply:
I just read the orginal comment DI left, but I think that was more about the lack of `design` in the name (in the original PR the package name was `kaizen-tokens`), rather than the specifically the addition of `kaizen` int the name.

Reply:
I haven't delved in depth for ramifications, but other ideas:
- less and scss could end up in the same directory to avoid needing `sass` directory
- could use symlinks to shortcut the deep folder heirachy for a nicer API. That might mess with tooling though.

It be good if we could make the imports less verbose (and more like what people are currently used to)

Reply:
That's right @mbylstra, I was referring to using `design-tokens` to match industry naming instead of `tokens`, and have no strong feelings on how we include `kaizen`.